### PR TITLE
Add support for customizing default look of plots.

### PR DESCRIPTION
**Pull request by syamajala**

### CODE REVIEW

File: ami/flowchart/library/Editors.py [hunk 1]
```diff
@@ -118,7 +139,7 @@ def restoreState(self, state):
 class HistEditor(TraceEditor):
 
     def __init__(self, parent=None, **kwargs):
-        kwargs['uiTemplate'] = [('brush', 'color', {'value': kwargs.get('color', (255, 0, 0))})]
+        kwargs['uiTemplate'] = [('Brush', 'color', {'value': kwargs.get('color', (255, 0, 0))})]
```
Comment (syamajala) [inline ami/flowchart/library/Editors.py:142]: This was a typo. In order to make the stylesheet consistent I changed it to Brush from brush. This means existing plots that customized the histogram color will lose the color unless you edit the fc file and change it from brush to Brush, or you could just reset the color in the editor manually.

File: examples/stylesheet.json [hunk 2]
```diff
@@ -0,0 +1,84 @@
+{
```
Comment (syamajala) [inline examples/stylesheet.json:1]: Here is an example stylesheet. Put it in ~/.config/ami.

Comment (ddamiani) [reply examples/stylesheet.json:1]: This looks good!

**Review by vespos — APPROVED** [review]
Looks good to me too!

**Pull request merged**